### PR TITLE
fix typos in test/e2e/kubectl/kubectl.go

### PR DESCRIPTION
**Pull request by k-toyoda-pi**
**What this PR does / why we need it**:

This is my first PR. I have checked all files in test/e2e/kubectl directory, and I found some typos.
So I fix typos in test/e2e/kubectl/kubectl.go. Also, change description for 'kubectl run' properly.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/kubectl/kubectl.go [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ metadata:
 		/*
 			Release : v1.9
 			Testname: Kubectl, label update
-			Description: When a Pod is running, update a Label using ‘kubectl label’ command. The label MUST be created in the Pod. A ‘kubectl get pod’ with -l option on the container MUST verify that the label can be read back. Use ‘kubectl label label-’ to remove the label. Kubetctl get pod’ with -l option SHOULD no list the deleted label as the label is removed.
+			Description: When a Pod is running, update a Label using ‘kubectl label’ command. The label MUST be created in the Pod. A ‘kubectl get pod’ with -l option on the container MUST verify that the label can be read back. Use ‘kubectl label label-’ to remove the label. ‘kubectl get pod’ with -l option SHOULD no list the deleted label as the label is removed.
```
Comment (oomichi) [inline test/e2e/kubectl/kubectl.go]: "SHOULD no list" also seems strange.
How about fixing it also in this PR?

Comment (k-toyoda-pi) [reply test/e2e/kubectl/kubectl.go]: Thank you for your reply.
I changed "SHOULD no" to "SHOULD not". Could you review it?

**Review by oomichi — COMMENTED** [review]
Thanks for updating

/lgtm

**Comment by k-toyoda-pi** [comment]
/assign @liggitt
 for approval

**Comment by liggitt** [comment]
/approve

please squash to a single commit before merge

**Comment by k-toyoda-pi** [comment]
@liggitt Thank you for the approval. I have squashed 2 commits.

**Pull request merged**